src/components/tests/Button.test.js

```diff
@@ -0,0 +1,25 @@
+// dependencies
+import React from 'react';
+import renderer from 'react-test-renderer';
+import { render, screen, cleanup } from '@testing-library/react';
+// local files
+import Button from '../Button';
+
+afterEach(() => {
+  cleanup();
+});
+
+describe('Button Component renders correctly', () => {
+  const buttonText = 'hello world';
```

Comment:
add lines 13-16 directly in the `test('should render button with proper text')` block, as they are not used for the snapshot. variables/implementation should be relative to the function block where they are used.

Reply:
For this one, moved everything except buttonText, as the snapshot uses this variable. Let me know if this is okay - thanks